src/Umbraco.Web/Models/ContentEditing/ContentTypeBasic.cs

```diff
@@ -48,7 +48,7 @@ public string IconFilePath
             {
                 return IconIsClass
                            ? string.Empty
-                           : IOHelper.ResolveUrl("~/umbraco/images/umbraco/" + Icon);
+                           : IOHelper.ResolveUrl("images/umbraco/" + Icon);

```

Comment:
What needs to be changed here is the 'umbraco' path needs to come from the GlobalSettings